.happy/terraform/modules/sfn/main.tf

```diff
@@ -160,7 +160,7 @@ resource "aws_sfn_state_machine" "state_machine" {
                     ]
                   }
                 },
-                "TimeoutSeconds": 36000
+                "TimeoutSeconds": 72000
```

Comment:
Why not increase to 360000 to match cxg conversion? The job is a multi-day job so I worry even 20 hours could stall in the queue. 

Also, we should up DownloadValidate as well, there was 1 instance of it timing out on 36000 as well

Reply:
I don't think we want a single upload job to take 10 days. We should instead allocate more machines to get the job done sooner.

Reply:
I will increase the timeout for download validate to match.

Reply:
We could also reduce the cxg processing time to 5 days.

Reply:
The cxg timeout is already about 4 days (360000 seconds == 100 hours == a little over 4 days) 

An upload job could still take multiple days because of the cxg processing timeout, so I don't think it affects much to have other jobs match. Though I agree we should look to increase resources; I'm just not sure what the 'right' amount would be to offset a 20 hour timeout, and I'd rather avoid timeouts in prod at all